### ISSUE TITLE
Not complete

### ISSUE THREAD

**Issue by tondemut** [samakshjain/thenewboston-django#1]
The django app is incomplete.

**Comment by samakshjain** [comment]
@tondejmut Yes, its not complete. And I have no time to get back to it. Maybe fork it and you can continue with the tutorial.
